Source/AwsCommonRuntimeKit/http/HttpStreamCalbackCore.swift

```diff
@@ -52,11 +52,9 @@ private func onResponseHeaders(stream: UnsafeMutablePointer<aws_http_stream>?,
     var headers = [HttpHeader]()
```

Comment:
utterly trivial: this filename misspells "Calback"

this could just be a TODO

Reply:
Thanks, I will fix it as part of renaming PR.